doc/classes/ConfigFile.xml

```diff
@@ -51,7 +51,7 @@
 		var err = config.load("user://scores.cfg")
 
 		# If the file didn't load, ignore it.
-		if err != OK:
+		if err == not OK:
```

Comment:
This is not correct, the style guide says to use:
```gdscript
if not X:
```
Instead of 
```gdscript
if !X:
```

This is different

This is not a valid change